recipe/meta.yaml

```
@@ -10,26 +10,39 @@ source:
   sha256: 3bf76a66361242b66adbecf332071b887143bf82f2ab7a3aabee5e54d7cdad09
 
 build:
-  noarch: python
-  number: 0
-  script: python -m pip install . -vv
-
-requirements:
-  host:
-    - python >=3.6
-    - pip
-  run:
-    - python >=3.6
-    - rdkit
-
-test:
-  imports:
-    - cmiles
-  requires:
-    - pip
-  commands:
-    - pip check
-    - python -c "import cmiles; print(cmiles.__version__)"
+  number: 1
+
+outputs:
+  - name: cmiles-base
+    build:
+      script: python -m pip install --no-deps .
+      noarch: python
+
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+      run:
+        - python >=3.6
+
+    test:
+      requires:
+        - rdkit
+      imports:
+        - cmiles
+
+  - name: cmiles
+    build:
+      noarch: python
+
+    requirements:
+      run:
+        - rdkit
+        - cmiles-base
```

Following matplotlib's [example](https://github.com/conda-forge/matplotlib-feedstock/blob/master/recipe/meta.yaml#L80), this might need to be: 

```suggestion
        - {{ pin_subpackage('cmiles-base', max_pin="x.x.x") }}
```

---

To be honest, I saw this in MPL's recipe and a) was spooked by it and b) though it might be overkill here. Giving it a spin to see what happens

---

I'm pinning to `x.x.x` for now, but I don't think it'll matter until this package is updated a few times